onnx_tf/backend_tf_module.py

```diff
@@ -20,61 +20,34 @@ def __init__(self, handlers, opset, strict, graph_def, backend):
     self.initializer_dict = self._get_initializer_from_graph_and_subgraphs(
         self.graph_def, dict())
     self.handler_variables = self._create_handlers_variables(
-        self.graph_def, dict())
+        self.graph_def, self.initializer_dict, dict())
 
   # get initializer from the main graph and all subgraphs in loop or if or scan
   # into tensor_dict
-  def _get_initializer_from_graph_and_subgraphs(self, graph, graph_tensor_dict):
-    if graph.initializer:
-      graph_tensor_dict.update(
-          self.backend._onnx_initializer_to_input_dict_items(graph.initializer))
-    for node in graph.node:
-      if node.op_type in ['Loop', 'Scan']:
-        onnx_node = OnnxNode(node)
-        body = onnx_node.attrs["body"]
-        graph_tensor_dict = self._get_initializer_from_graph_and_subgraphs(
-            body, graph_tensor_dict)
-      elif node.op_type == 'If':
-        onnx_node = OnnxNode(node)
-        then_branch = onnx_node.attrs['then_branch']
-        graph_tensor_dict = self._get_initializer_from_graph_and_subgraphs(
-            then_branch, graph_tensor_dict)
-        else_branch = onnx_node.attrs['else_branch']
-        graph_tensor_dict = self._get_initializer_from_graph_and_subgraphs(
-            else_branch, graph_tensor_dict)
-    return graph_tensor_dict
+  def _get_initializer_from_graph_and_subgraphs(self, graph, init_dict):
+    if self.handlers:
+      if graph.initializer:
```

Comment:
Check for self.handlers is not necessary. There must be handlers, or nothing we can do anyway.